doc/users/verify_quote.rst

```diff
@@ -1,33 +1,34 @@
 Verifying Quote
 ===============
 
-A client can verify the SGX quote of the CCF node that it connects to.
-
-First, the client should connect to the node to verify, specifying the ``/node/quote`` endpoint:
+A client can verify the validity of the SGX quote of the CCF node that it connects to. This can be done using the ``verify_quote.sh`` script installed as part of the CCF install or ccf Python package (see :doc:`/quickstart/install`).
 
 .. code-block:: bash
 
-    $ curl https://<ccf-node-address>/node/quote --cacert networkcert.pem
-    {"quotes": [{"mrenclave":"<measurement_hash>, "node_id":<node_id>, "raw":"<hex_encoded_raw_quote>"}]}
+    $ verify_quote.sh https://<ccf-node-address> [--mrenclave <mrenclave_hex>] [CURL_OPTIONS]
+    Retrieved 1 accepted code versions from CCF service.
+    Node quote successfully retrieved. Verifying quote...
+    Quote verification successful.
+
+The script verifies the target CCF node's SGX quote by:
+
+1. Retrieving the node's SGX quote via the ``/node/quote`` endpoint.
+2. Verifying that the cryptographic hash of the node's identity public key (as presented in the node's certificate in the TLS session) matches the quote's report data.
+3. Verifying that the quote's measurement (``MRENCLAVE``) matches one of the trusted enclave measurements.
 
-The raw quote should be decoded and output to file for verification via the Open Enclave ``oeverify`` command-line utility:
+A specific trusted SGX enclave measurement can be specified ``--mrenclave <mrenclave_hex>``.
```

Comment:
Maybe add a reference to how to run `oesign -dump` to extract mrenclave for an enclave file?